### PR TITLE
Adds upload-override-jars script.

### PR DESCRIPTION
**Pull request by remexre**
# Changes

Adds a script for uploading JARs to Jenkins for use with `OVERRIDE_JARS`.

# Documentation

Does this need docs?

### CODE REVIEW

File: upload-override-jars [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+
+if [[ -n "$(git status --porcelain)" ]]; then
+	echo "You have uncommitted changes! Refusing to run."
+	exit 1
+fi
+
+branch_name="$(git rev-parse --abbrev-ref HEAD)"
+commit_hash="$(git rev-parse HEAD)"
+
+rsync -Pa jars/ foundry.remexre.xyz:/melt/jenkins/export-scratch/melt-jenkins/${commit_hash}-jars/
```
Comment (krame505) [inline upload-override-jars:15]: Not super familiar with rsync, but will this work if the user from which the script is run doesn't happen to match the user on foundry?

Comment (remexre) [reply upload-override-jars:15]: One can provide a username to use via `~/.ssh/config` (rsync just uses SSH under the hood)

Comment (krame505) [reply upload-override-jars:15]: You could maybe add a comment to the script mentioning that.  I guess it's maybe not a huge issue anyway as only melt people should ever run this, and we can pick the account names on foundry.

Comment (remexre) [reply upload-override-jars:20]: Added.

**Review by krame505 — APPROVED** [review]

**Pull request merged**